.gitignore

```diff
@@ -159,3 +159,5 @@ napari/settings/napari.schema.json
 docs/jupyter_execute/
 docs/.jupyter_cache/
 docs/gallery/
+
+report*.json
```

Comment:
I have a vague memory of where these `report*.json` files come from, but can't remember right. I really like having a comment above each block in the `.gitignore`, so it would be great if you could add those.

Reply:
from #4518 Something like "pytest reports in json format"?

Reply:
Perfect! The link to the PR could also be useful.